OrbitQt/CallTreeWidget.h

```diff
@@ -46,6 +46,65 @@ class CallTreeWidget : public QWidget {
   static const QString kActionDisassembly;
   static const QString kActionCopySelection;
 
+  template <
+      typename HideValuesProxyModelT,
+      typename = std::enable_if_t<std::is_base_of_v<QIdentityProxyModel, HideValuesProxyModelT>>>
+  void SetCallTreeView(std::unique_ptr<CallTreeView> call_tree_view) {
+    CHECK(app_ != nullptr);
+
+    model_ = std::make_unique<CallTreeViewItemModel>(std::move(call_tree_view));
+
+    hide_values_proxy_model_ = std::make_unique<HideValuesProxyModelT>(nullptr);
+    hide_values_proxy_model_->setSourceModel(model_.get());
```

Comment:
Have you considered passing in this proxy model as an argument instead of constructing it here? This would mean the `HideValues`-proxy models get implementation details and can be moved to the cpp file. The template here is probably also not needed in this case. But maybe I'm missing something. :see_no_evil:

Reply:
You're right, for some reason I was assuming I was setting the underlying model during construction, but that obviously happens with `setSourceModel`. Done.